src/schemathesis/runner/impl/solo.py

```diff
@@ -15,6 +15,8 @@ class SingleThreadRunner(BaseRunner):
     """Fast runner that runs tests sequentially in the main thread."""
 
     request_tls_verify: Union[bool, str] = attr.ib(default=True)  # pragma: no mutate
+    request_cert: str = attr.ib(default=None)  # pragma: no mutate
```

Comment:
I think that these attributes should have the `Optional[str]` type

Reply:
Fixed.